jkube-kit/resource/service/src/test/java/org/eclipse/jkube/kit/resource/service/DefaultResourceServiceTest.java

```diff
@@ -60,22 +55,22 @@ public class DefaultResourceServiceTest {
   private ResourceServiceConfig resourceServiceConfig;
   private DefaultResourceService defaultResourceService;
 
-  @Before
-  public void init() throws IOException {
-    targetDir = temporaryFolder.newFolder("target");
+  @BeforeEach
+  void init(@TempDir File temporaryFolder) {
+    targetDir = new File(temporaryFolder, "target");
     resourceServiceConfig = ResourceServiceConfig.builder()
         .interpolateTemplateParameters(true)
         .targetDir(targetDir)
         .project(project)
         .resourceFileType(ResourceFileType.yaml)
-        .resourceDirs(Collections.singletonList(temporaryFolder.newFolder("resources")))
+        .resourceDirs(Collections.singletonList(new File(temporaryFolder, "resources")))
```

Comment:
We should use an actual directory, not just a reference to a nonexistent one

Reply:
Got it